criu/page-pipe.c

```diff
@@ -99,6 +99,7 @@ static struct page_pipe_buf *ppb_alloc(struct page_pipe *pp, unsigned int ppb_fl
 {
 	struct page_pipe_buf *prev = pp_prev_ppb(pp, ppb_flags);
 	struct page_pipe_buf *ppb;
+	int ppb_size = 0;
```

Comment:
nit: s/int/long/

Reply:
The return type of the fcntl function is int

Reply:
The return type of the system call is long: https://elixir.bootlin.com/linux/latest/source/fs/fcntl.c#L442

Reply:
you are right, the return type of the libc wrapper is int. Moreover, the pipe size is limited by (1<<31): https://elixir.bootlin.com/linux/latest/source/fs/pipe.c#L1236